extensions/walletProviders/utxo/README.md

```diff
@@ -0,0 +1,104 @@
+# 🌍 UTXO Wallet Provider
+
+UTXO Wallet Provider integrates seamlessly with Tatum SDK to provide extended wallet capabilities for UTXO-based blockchains.
+
+## 📖 Description
+
+The UTXO Wallet Provider provides an array of tools for:
+
+- Generating mnemonics for seed phrases.
+- Creating extended public keys (xpubs) from mnemonics.
+- Deriving private keys and addresses from mnemonics and xpubs.
+- Signing and broadcasting transactions to the UTXO-based networks.
+
+It is built upon popular packages like `bitcoinjs-lib`, `bitcore-lib`, `bip39`, and `bip32`, ensuring a robust and secure foundation.
+
+## 🚀 Quick Start
+
+1. **Installation**
+
+   Firstly, ensure that the `@tatumio/utxo-wallet-provider` package is set as a dependency within your project. Next, import the UTXO Wallet Provider extension:
+
+   ```typescript
+   import { UtxoWalletProvider } from '@tatumio/utxo-wallet-provider';
+   ```
+
+2. **Initialization**
+
+   Create an instance of Tatum SDK passing `UtxoWalletProvider` as one of wallet providers.
+
+   ```typescript
+   const tatumSdk = await TatumSDK.init<Bitcoin>({
+        network: Network.BITCOIN,
+        configureWalletProviders: [
+            UtxoWalletProvider,
+        ]
+    })
+   ```
+
+## ⚙️ Configuration
+
+   You can configure the `UtxoWalletProvider` as below to have all the checks skipped. This is useful for development and testing purposes.
+
+   ```typescript
+   const tatumSdk = await TatumSDK.init<Bitcoin>({
+        network: Network.BITCOIN,
+        configureWalletProviders: [
+           {type: UtxoWalletProvider, config: {skipAllChecks: true}},
```

Comment:
not sure about skipAllChecks here, it was for estimation from core api to sdk, during tx preparation. it looks like not so global config. maybe we can move it somewhere or maybe even delete, not sure it is needed for user

Reply:
Well, it is exposed and used by the bitcore itself, apparently for development purposes...